starter_code/src/movies.js

```diff
@@ -4,9 +4,27 @@
 
 // Get the average of all rates with 2 decimals 
 
+function ratesAverage(moviesArray) {
 
-// Get the average of Drama Movies
+    moviesRate = moviesArray.map(function (oneMovie) {
+        return parseFloat(oneMovie.rate);
+    });
+
+    rateTotal = moviesRate.reduce(function (sum, oneRate) {
+        return sum + oneRate;
```

Comment:
For Number to work and pass the tests, you need to add an extra set of parentheses so that it includes the .toFixed()

` return Number((rateTotal / moviesArray.length).toFixed(2));`